src/items/FilePdf.tsx

```diff
@@ -41,11 +43,13 @@ const FilePdf: FC<FilePdfProps> = ({
     }
   };
 
+  const urlWithPdfViewer = `${PDF_VIEWER_LINK}${encodeURIComponent(url)}`;
```

Comment:
We should probably perform some sanitization here. Maybe we could ensure that url starts with our domain?
Validation should be performed on the PDF viewer side, to ensure that malicious users cannot load arbitrary content or use our PDF viewer as a CDN.